lib/repository/repository.ts

```diff
@@ -152,8 +149,11 @@ export default abstract class Repository<TEntity extends Entity> {
    * @param ids The IDs of the {@link Entity | Entities} you seek.
    * @returns The matching Entities.
    */
-   async fetchMany(...ids: string[]): Promise<TEntity[]> {
-    return await this.readEntities(ids);
+  async fetch(...ids: string[]): Promise<TEntity[]>
+
+  async fetch(...id: string[]): Promise<TEntity | TEntity[]> {
```

Comment:
Does this need to be marked as internal for TypeDoc?

Reply:
I doubt it because they are overloads, but i never used typedoc so i cant comment to much. I wouldnt tho because if JSDoc grabs the internal the overloads wont work

Reply:
Just a quick `npm run docs` and then look in the docs folder.

Reply:
Ok so, @internal doesnt seem to affect typescript but i dont think its a good idea to use it, because on type doc it only shows the overloads and not the final version, so it might be better to keep it
Screenshot without the @internal:
![image](https://user-images.githubusercontent.com/80978939/176028692-a2d9b467-2cb7-4ae6-a1f6-886e516fb1a5.png)